### ISSUE TITLE
problem with upgrade ESP8266 to ESP32 (EMS-ESP GATEWAY V 1.3)

### ISSUE THREAD

**Issue by lsw271** [emsesp/EMS-ESP32#794]
Hello, 
I've got a problem with upgrading my EMS-ESP Gateway (v 1.3) with ESP32. I'm using "MH-ET Live D1". I did all like in documentation (https://bbqkees-electronics.nl/wiki/gateway/upgrade-the-gateway-to-an-esp32.html) and i've got a problem with bus connection. I tried connect with jack but in web application I saw "EMSbus disconnected, check settings and board profile". I tried to change profiles but always the same result. So I thought it will be better to connect with USB and EMS directly in the boiler (without jack), but it was the same message in web. 
I used EMS-ESP FLASHER. 
Red led is always on, there also blue led. Blue led is blinking then I got two seconds delay, again blinking blue etc.
Yesterday I could receive information by mqtt from heartbeat (WIFi signal, memory etc.) but sometims web interface was unavailable - the website was blank, only the icon in browse tab and description "EMS-ESP". Now it looks reseting after few minutes of working, connect to wifi and mqtt and reset again but when it's online I can't get to web interface. I just see information in Home assistant
LAST STABLE VERSION 3.4.4

Do you have some ideas what should I do now to get working EMS-ESP GATEWAY?

**Comment by MichaelDvP** [comment]
Some thing to check:
- For USB-power pull the DCDC board from gateway. 
- Check soldering of 3.3V, RX, TX pins (from USB side view the 3 lower pins left.
- Measure 3.3V on gateway (J20)
- For Jack-power also measure 5V on gateway and ESP-pin. Adjust DCDC is needed,

**Comment by bbqkees** [comment]
Looks like a power issue of some sort.
On these older boards always remove the green voltage regulator board and use USB to power it all.

Is the ESP32 board in the right position on the Gateway board?
It is not flipped around or mounted backwards?

Can you disconnect the ESP32 from the Gateway board, connect it to USB and see if it ESP32 runs stable or not for a day or so?

Also make sure the right pins are connected to TX and RX in the board profile settings.


If you can't make it work in the end; Existing owners of an ESP8266 Gateway can upgrade to a new Gateway (S32 or E32) with a 30% discount.

**Comment by lsw271** [comment]
Yes, I removed green voltage regulator. 
I think its the right position. I tried also connect ESP to board with wires - only 5wires (3.3v / RX / TX / 5v / GND) the result was the same.
Now my ESP its connect only to USB power without gateway and its working since 46hours (I see heartbeat in HA sends by mqtt). But sometimes I can't get to ESP website - now website is white / blank. Yesterday I can login and I saw everything.

I didn't checked yet what Michael suggest - I will try on Saturday
